Source/CParse/parser.y

```diff
@@ -2043,66 +2043,50 @@ clear_directive : CLEAR tm_list SEMI {
    ------------------------------------------------------------ */
 
 constant_directive :  CONSTANT identifier EQUAL definetype SEMI {
-		   if ($4.type != T_ERROR) {
-		     SwigType *type = NewSwigType($4.type);
-		     $$ = new_node("constant");
-		     Setattr($$,"name",$2);
-		     Setattr($$,"type",type);
-		     Setattr($$,"value",$4.val);
-		     if ($4.rawval) Setattr($$,"rawval", $4.rawval);
-		     Setattr($$,"storage","%constant");
-		     SetFlag($$,"feature:immutable");
-		     add_symbols($$);
-		     Delete(type);
-		   } else {
-		     Swig_warning(WARN_PARSE_UNSUPPORTED_VALUE,cparse_file,cparse_line,"Unsupported constant value (ignored)\n");
-		     $$ = 0;
-		   }
-
+		 SwigType *type = NewSwigType($4.type);
+		 $$ = new_node("constant");
+		 Setattr($$,"name",$2);
```

Comment:
Suggest whitespace fixes (spaces after the comma) given other whitespace indentation change on this line and others.